bigquery/samples/client_query_legacy_sql.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def client_query_legacy_sql(client):
+
+    # [START bigquery_query_legacy]
+    from google.cloud import bigquery
+
+    # TODO(developer): Construct a BigQuery client object.
+    # client = bigquery.Client()
+
+    query = (
+        "SELECT name FROM [bigquery-public-data:usa_names.usa_1910_2013] "
+        'WHERE state = "TX" '
+        "LIMIT 100"
+    )
+
+    # Set use_legacy_sql to True to use legacy SQL syntax.
+    job_config = bigquery.QueryJobConfig()
+    job_config.use_legacy_sql = True
+
+    # Start the query, passing in the extra configuration.
+    query_job = client.query(
+        query,
+        location="US",  # Must match the source and the destination dataset(s) location.
```

Comment:
Query location detection works pretty well now. Let's delete this line.

```suggestion
```


Also we should delete it from most of the other query samples in a different PR. I've filed https://github.com/googleapis/google-cloud-python/issues/9499 to track this work.